cvmfs/network/jobinfo.h

```diff
@@ -126,6 +169,8 @@ class JobInfo {
   shash::ContextPtr *GetHashContextPtr() { return &hash_context_; }
   Pipe<kPipeDownloadJobsResults> *GetPipeJobResultWeakRef() {
                                            return pipe_job_results.weak_ref(); }
+  Tube<DataTubeElement> *GetDataTubeWeakRef() {
+                                                 return data_tube_.weak_ref(); }
```

Comment:
I think `weak_ref` is the wrong word in the `UniquePtr` class to begin with. But let's not perpetuate that.
```suggestion
  Tube<DataTubeElement> *GetDataTubePtr() {
                                                 return data_tube_.weak_ref(); }
```

Reply:
i oriented myself at the pipe stuff... i will rename both of them